x-pack/plugins/security_solution/common/api/timeline/persist_note/persist_note_route.ts

```diff
@@ -19,3 +19,22 @@ export const persistNoteSchema = runtimeTypes.intersection([
     version: unionWithNullType(runtimeTypes.string),
   }),
 ]);
+
+export const persistNoteWithoutRefSchema = runtimeTypes.intersection([
```

Comment:
You may need to update this ref: https://github.com/elastic/kibana/blob/db5ff403594ecb833137454a09e9455c18de740e/x-pack/plugins/security_solution/common/api/timeline/persist_note/persist_note_route_schema.yaml#L33


Eventually the plan is to auto generate the types from this open api schema and use zod in place of io-ts